src/atoms/atomgraph.jl

```diff
@@ -127,9 +79,8 @@ Construct an AtomGraph object from a structure file.
 """
 function AtomGraph(
     input_file_path::String,
-    id::String = "",
+    id::String = splitext(input_file_path)[begin],
```

Comment:
sorry can we actually do this as `splitext(splitpath(input_file_path)[end])[begin]`? i.e. just the file name, not the beginning of the path or the extension

Reply:
Is it possible that you'd want to build `AtomGraph`s from two files that have the same file name, but reside in different directories in a batch? If it is, then this might make it difficult to distinguish between the two `AtomGraph`'s (if required).
I wasn't entirely sure about that, so I thought I'd leave it as-is 😅

Reply:
I mean, this is just for the default `id` value. Obviously the user can specify themselves if they want something else. I suppose a future option we could consider would be something like specifying a prefix and otherwise using the default, but I don't think that's necessary right now...anyway, I'm fine leaving this as-is too, it's not a big deal, just wanted some sensible default in there that was more useful than the empty string, which I put in ages ago and forgot to change :P